internal/dag/httpproxy_processor.go

```diff
@@ -225,8 +225,14 @@
 				}
 
 				svhost.AuthorizationService = ext
-				svhost.AuthorizationResponseTimeout = timeout.Parse(auth.ResponseTimeout)
 				svhost.AuthorizationFailOpen = auth.FailOpen
+
+				timeout, err := timeout.Parse(auth.ResponseTimeout)
```

Comment:
Nit: Maybe a test for this one? No problems if not, because I'm going to have to refactor all the status tests anyway.

Reply:
ah yeah, I think I breezed over that during a rebase. Added one.